jobs/rocoto/ocnpost.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+set -x
+
+###############################################################
+## CICE5/MOM6 post driver script 
+## FHRGRP : forecast hour group to post-process (e.g. 0, 1, 2 ...)
+## FHRLST : forecast hourlist to be post-process (e.g. anl, f000, f000_f001_f002, ...)
+###############################################################
+
+# Source FV3GFS workflow modules
+. $HOMEgfs/ush/load_fv3gfs_modules.sh
```

Comment:
I don't see this script in the new branch.

Reply:
Which script? ocnpost.sh or load_fv3gfs_module.sh? Both are in my branch:
https://github.com/KateFriedman-NOAA/global-workflow/blob/feature/coupled_sprint/jobs/rocoto/ocnpost.sh
https://github.com/KateFriedman-NOAA/global-workflow/blob/feature/coupled_sprint/ush/load_fv3gfs_modules.sh

> Has this been tested or is this just to copy over scripts?

Just a copy right now. The ocnpost jobs have to be added to the setup_workflow*py scripts still before they can be tested. I can run an atmos-only free-forecast test to be sure nothing is broken...let me know.

Reply:
Must need more coffee, I somehow didn't see the load_fv3gfs_modules in your branch before.